### ISSUE TITLE
How to Cntrl-Delete?

### ISSUE THREAD

**Issue by stefandrissen** [simonowen/simcoupe#95]
For obscure reasons I am using Comet in SimCoupe and it is driving me nuts.

In Comet to delete a line I need to use Cntrl-Delete, but how do I do this? Ctrl+Del does not work, since as the Wiki indicates, the Delete key is mapped to Shift+Backspace.

**Comment by stefandrissen** [comment]
Never mind, found it, the secret handshake is right Ctrl + Backspace  :-)

**Comment by simonowen** [comment]
I was thinking there might be more going on, but it's just that Backspace is mapped to SAM's Delete key, and Right-Ctrl is mappped to SAM's Cntrl key. So Right-Ctrl+Backspace is the pure key combination for that Comet shortcut. The PC keys are mapped to their expected PC *function* rather than name, which unfortunately means it's reversed compared to the SAM keyboard.

I do have some special cases where both shift keys count individually. The PC `=` key is shifted to give `+` on my UK keyboard. In SimCoupe if you press both shift keys and `=` you get `*` instead. That's because Shift and `=` key gets the `+` symbol. That's on an unshifted SAM key, so the first shift is consumed and released. The second shift works like a plain shift key and the SAM shifts `+` into `*` on the same key. I very rarely use that feature but it seemed like a good idea at the time!

I did think Shift+Right-Ctrl+Delete might work to cancel the hidden shift on the Delete->Shift-Backspace key combination, giving what you want. It doesn't work at the moment but I might add it so it works like that, to give similar special shift logic to what I described above.
